crates/router/src/core/payments/transformers.rs

```diff
@@ -513,7 +514,18 @@ where
                 });
 
                 let amount_captured = payment_intent.amount_captured.unwrap_or_default();
-                let amount_capturable = Some(payment_attempt.amount - amount_captured);
+                let should_calculate_amount_to_capture = |intent_status| {
+                    matches!(
+                        intent_status,
+                        IntentStatus::PartiallyCaptured | IntentStatus::RequiresCapture
+                    )
+                };
+                let amount_capturable = if should_calculate_amount_to_capture(payment_intent.status)
```

Comment:
this data will be available on the parent payment_attempt as amount_to_capture right?

Can we use that instead calculating it again?.